test/unit/step-value-test.js

```diff
@@ -0,0 +1,52 @@
+'use strict';
+
+const assert = require('assert');
+const sinon  = require('sinon');
+const StepValue = require('../../lib/step-value');
+
+describe('StepValue', function() {
+  let stepValue, compiledStep, matchingDefinitions;
+
+  beforeEach(function() {
+    compiledStep = {text: 'I am the step text', keyword: 'Given'};
+    matchingDefinitions = [
+      { implementation: sinon.spy() }
+    ];
+    stepValue = new StepValue(compiledStep, matchingDefinitions);
+  });
+
+  it('makes available the most important parts of the step', function() {
+    assert.equal(stepValue.text, 'I am the step text');
+    assert.equal(stepValue.type, 'Given');
+    assert.equal(stepValue.implementation, matchingDefinitions[0].implementation);
```

Comment:
You would probably want this to be a `deepEqual`, since this just asserts that the references are the same. Unless you are wanting to track you definition references, while passing them around.